datumaro/plugins/yolo_format/converter.py

```diff
@@ -110,9 +110,9 @@ def apply(self):
 
             for subset_name, subset_list_name in subset_lists.items():
                 f.write('%s = %s\n' % (subset_name,
-                    osp.join('data', subset_list_name)))
```

Comment:
follow instruction from AlexeyAB to create dataset will have the folder structure below
```bash
├── data
│   ├── train.txt
│   ├── obj.name
│   ├── obj.data
│   ├── obj
│   │   ├── img1.jpg
│   │   ├── img1.txt
│   │   ├── img2.jpg
└─└─└── img2.txt
```

for now datumaro generate yolo dataset folderstructure like this
```bash
├── merged
│   ├── train.txt
│   ├── obj.name
│   ├── obj.data
│   ├── obj_train_data
│   │   ├── img1.jpg
│   │   ├── img1.txt
│   │   ├── img2.jpg
└─└─└── img2.txt
```
so the "data" in path is wrong and should be remove